js/client-battle-tooltips.js

```diff
@@ -1152,6 +1165,47 @@ var BattleTooltips = (function () {
 			basePower = Math.floor(basePower * abilityBoost);
 			basePowerComment = this.makePercentageChangeText(abilityBoost, ability);
 		}
+
+		var allyActive = pokemon.side.active;
+		var auraBoosted = '';
+		var auraBroken = false;
+		if (move.category !== 'Status' && allyActive.length > 1) {
+			for (var i = 0; i < allyActive.length; i++) {
+				var ally = allyActive[i];
+				if (!ally || ally.fainted) continue;
+				if (!auraBoosted && (ally.ability === 'Fairy Aura' && move.type === 'Fairy') ||
+							(ally.ability === 'Dark Aura' && move.type === 'Dark')) {
```

Comment:
I actually don't know what the convention is here. I think Cathy wanted to do +2 tabs, but I think consensus is on +1 tab?

Reply:
+3 tabs is right out, though.

Reply:
Might as well delete the ``!auraBoosted`` check and there is no longer a debate to have.
That would also fix the ``&&`` vs ``||`` precedence issue.